zrml/court/src/lib.rs

```diff
@@ -878,16 +887,31 @@ mod pallet {
 
                     let raw_vote =
                         Vote::Revealed { commitment, vote_item: vote_item.clone(), salt };
-                    draws[index] = Draw { court_participant: who.clone(), vote: raw_vote, ..draw };
+                    let slashable_amount = draw.slashable;
+                    let draw_weight = draw.weight;
+                    draws[index] = Draw {
```

Comment:
We seem to have missed panicker here. Accessing a bounded vector via `Index` can panic if `index` is out of range.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1182/commits/6e0574a91f15e85bcbeadeeb6c4482ee808bafd7